cmd/group/main.go

```diff
@@ -38,7 +40,8 @@ func main() {
 
 	logLevel := len(log.AllLevels) - 2
 
-	listen := "unix:///run/infrakit/plugins/group.sock"
+	listen := "unix:///run/infrakit/plugins/"
+	sock := "group.sock"
```

Comment:
Is there a reason for splitting this into two separate flags?   The url (with `unix://` as the protocol/scheme) seems to be more common for specifying listen address.   Also, you end up joining the strings to make a url later on in the code.